File: interview_code/fizzbuzz.rb
def fizzbuzz(upper_limit)
    if !upper_limit.is_a? Integer
        puts "Please use an integer for an argument"
    else
        for i in 1..upper_limit
            if  i%3 == 0 && i%5 == 0
                puts "fizzbuzz"
            elsif i%3 == 0
                puts "fizz"
            elsif i%5 == 0
                puts "buzz"
            else 
                puts i
            end
        end
    end
end

fizzbuzz(50)

File: interview_code/fizzbuzz.js
// Completed, not refactored

function fizzbuzz(upperLimit) {
    document.body.innerHTML = "";
    for (var i = 1 ; i <= upperLimit; i++) {
        if( i%3 ===0 && i%5 ===0){
            console.log("fizzbuzz");
        } else if( i%3===0) {
            console.log("fizz");
        } else if( i%5===0) {
            console.log("buzz");
        } else {
            console.log(i);
        }
    }
        return "done";
}

fizzbuzz(50);